TESTING.md

```diff
@@ -85,7 +85,8 @@ An alternative to the MiniAccumuloCluster for testing, a standalone Accumulo clu
 most tests. This requires a manual step of building and deploying the Accumulo cluster by hand. The build can then be
 configured to use this cluster instead of always starting a MiniAccumuloCluster.  Not all of the integration tests are
 good candidates to run against a standalone Accumulo cluster, these tests will still launch a MiniAccumuloCluster for
-their use.
+their use.  They also take longer to run and it may be necessary to provide a timeout.factor property; `-Dtimeout.factor=3`
+is recommended.
```

Comment:
It's not clear which tests "They" refer to in "They also", or what they are being compared to in "longer to run".

In general, it's not true that running tests against a standalone cluster take longer to run. Usually, they're faster, because the tests avoid starting up a new MiniAccumuloCluster.

The `timeout.factor` has nothing to do with the standalone testing method. Rather, it has to do the performance of the machine on which the tests are being run. I would not recommend any specific value, since it depends so much on the user's test machine specs.

Instead of documenting this in the standalone testing section, it'd be better to mention this property in a section that applies equally to standalone testing and normal builds. Also, it could merely mention that some tests may take longer on certain hardware, and it may be useful to scale the normal timeouts accordingly with the `timeout.factor`. A value of 3 could be used as an example, but I don't think it should be recommended. (On my laptop, with 8GB RAM, an SSD, and a 5th gen Intel i7 mobile CPU, the most I've ever needed was a timeout factor of 2.)

Reply:
I'll remove the update from Testing.md.  
Just fyi - I'm running the uno large 32X4 (note that this helped with many of the timeout issues) on my laptop which has 32GB RAM with 12 cpu and 2 has often not been enough.   The uno standalone almost always takes longer running the IT tests.